packages/fxa-auth-server/test/remote/account_reset_tests.js

```diff
@@ -21,239 +22,194 @@ describe('remote account reset', function() {
     });
   });
 
-  it('account reset w/o sessionToken', () => {
+  it('account reset w/o sessionToken', async () => {
```

Comment:
Per https://github.com/mozilla/fxa/pull/2570#discussion_r327004747, I asyncified a bunch of the tests in this file, which was a nice cleanup but generated too much code churn that distracts from the review, so I stopped short of doing any more.  The important part of this change is the new `account reset updates keysChangedAt` test at the end of the file.